Jellyfin.Server/Migrations/Routines/DisableZealousLogging.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.IO;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Model.Configuration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Serilog;
+using ILogger = Microsoft.Extensions.Logging.ILogger;
+
+namespace Jellyfin.Server.Migrations.Routines
+{
+    /// <summary>
+    /// Updater that takes care of bringing configuration up to 10.5.0 standards.
+    /// </summary>
+    internal class DisableZealousLogging : IUpdater
+    {
+        /// <inheritdoc/>
+        public string Name => "DisableZealousLogging";
```

Comment:
```suggestion
        public string Name => "DisableOverzealousLogging";
```

Reply:
You might mean this instead.

Reply:
This is renamed `CreateUserLoggingConfigFile` in the next PR so the name here doesn't matter so much: https://github.com/jellyfin/jellyfin/pull/2535/files#diff-5559799524e9efb63701fb597cf293b9